pkg/module/metrics/drops.go

```diff
@@ -95,6 +98,11 @@ func (d *DropCountMetrics) ProcessFlow(flow *v1.Flow) {
 		return
 	}
 
+	if d.enableStandalone {
+		d.processStandaloneFlow(flow)
```

Comment:
We should split standard and standalone metrics module. Each mode should have its own enrichment/metric computation logic.